R/acs_var.R

```diff
@@ -14,3 +14,22 @@ unemp_race_codes <- c("S2301_C04_001",map_chr(seq(12,20),function(x)paste0("S230
 ed_race_codes <- map_chr(seq(28,54),function(x)paste0("S1501_C01_0",x)) # education attainment
 earnings_edu_codes <- map_chr(seq(1,6),function(x){paste0("B20004_00",x)}) # median earnings by education attainment
 
+# Transport ------------
```

Comment:
added codes for:
i.	Households without vehicles
ii.	Median earnings by race
iii.	Migration by education attainment